test/node/medtronic/testProcessData.js

```diff
@@ -320,7 +321,8 @@ describe('processData.js', function() {
           'resumeIndex' : 4,
     			'payload': {
     				'lgs_types': [
-    					'Suspend user',
+              'Suspend low glucose',
```

Comment:
Thanks, fixed it just now but forgot to push before merge. Cherry-picking into main branch.